.github/workflows/container-build.yml

```diff
@@ -0,0 +1,51 @@
+name: Build container image every change.
+
+on:
+  workflow_call:
+    outputs:
+      digest:
+        description: "Container image digest"
+        value: ${{jobs.build.outputs.digest}}
+  push:
+    branches:
+      - "*"
+
+jobs:
+  build:
+    name: Build
+    uses: kubewarden/policy-server/.github/workflows/container-image.yml@main
```

Comment:
This is taken from the kubewarden-controller, I don't know if we should force the usage of `@main` here (or in other places where we reuse a local (as in from the same repository) workflow.

I think we should instead use the `./.github/workflows/{filename}` syntax, because that ensures the workflow is taken from the same git rev of the "main" workflow.

See [this section](https://docs.github.com/en/actions/using-workflows/reusing-workflows) of the official docs:

> If you use the second syntax option (without `{owner}/{repo}` and `@{ref}`) the called workflow is from the same commit as the caller workflow.

Reply:
I've updated all the `uses` references inside of the GH actions to make use of the local file

Reply:
Agree with this change. Don't see any security issue, and at least it makes it easier to work with workflows over forks.